### ISSUE TITLE
SSR?

### ISSUE THREAD

**Issue by albinekb** [jaredpalmer/react-fns#31]
Hi! Just found this on trendig, love it 😍 

I'm using [next](https://github.com/zeit/next) (React with SSR), I have not yet tested this lib with SSR but since most of the exposed components from here needs browser APIs to function, there's different ways you could handle that in an SSR case.

Is SSR supported, if yes, it should be documented what the approach is (to not render on the server, sending placeholder values or something else)

Thanks!

**Comment by jaredpalmer** [comment]
Fair point. Should probably check for `typeof window === 'undefined` everywhere.

**Comment by prichodko** [comment]
Correct me if I am wrong, but since all the job is done in `componentDidMount`, which is called only on the client and not on the server, it should already work.

**Comment by jaredpalmer** [comment]
@prichodko this should be true, but might not be for Media. not 100% sure

**Comment by prichodko** [comment]
I tried it in Next and most of them are working. I had a problem with WindowSize (it throws `Cannot read property 'toLowerCase' of undefined`) and found potential bug in Network (will send PR).

**Comment by albinekb** [comment]
Would it be nice to have a SSR test? I can work on a PR.

**Comment by stereobooster** [comment]
It is possible to make `<Locales />` work on a server using headers from the request

**Comment by stereobooster** [comment]
`GeoPosition` can work on a server using client's IP and GeoIP DB. For sure this is out of scope for this repo
